homeassistant/components/telegram_bot/webhooks.py

```diff
@@ -33,9 +33,8 @@ async def async_setup_platform(hass, config):
     bot = initialize_bot(config)
 
     current_status = await hass.async_add_executor_job(bot.getWebhookInfo)
-    base_url = config.get(
-        CONF_URL, get_url(hass, require_ssl=True, allow_internal=False)
-    )
+    if not (base_url := config.get(CONF_URL)):
```

Comment:
Do we need the extra parentheses?
```suggestion
    if not base_url := config.get(CONF_URL):
```

Reply:
It's not a valid syntax without them

Reply:
Ok, I thought that it was.

https://www.python.org/dev/peps/pep-0572/#always-requiring-parentheses

Reply:
by itself it's valid, but since we are testing a negative here with the not operator, the parentheses is required

Reply:
Oh right, missed that.

Reply:
```python
>>> test_dict = {"test":"test"}
>>> if not test := test_dict.get("test"):
  File "<stdin>", line 1
    if not test := test_dict.get("test"):
       ^
SyntaxError: cannot use assignment expressions with operator
>>> if test := test_dict.get("test"):
...   pass
... 
>>> if not (test := test_dict.get("test")):
...   pass
... 
>>> 
```